src/components/Calculator.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import calculate from './logic/calculator';
+
+function Calculator() {
+  const [calc, calcUpdate] = useState({ total: null, next: null, operation: null });
+
+  const updateCalc = (e) => {
+    const currCalc = calculate(calc, e.target.value);
+    const display = document.querySelector('.display');
+    if (currCalc.total === null) {
+      currCalc.total = '0';
+    }
+    if (currCalc.total !== null) {
+      display.innerText = currCalc.next;
+    } else {
+      display.innerText = currCalc.total;
+    }
+    calcUpdate(currCalc);
+  };
```

Comment:
Let's fix the `updateCalc`, `result`, and `operators` seem not to show.

KIndly make sure the calculator is working correctly.